### ISSUE TITLE
[Buttons] incorrect accessibilityFrame

### ISSUE THREAD

**Issue by codeman7** [material-components/material-components-ios#7526]
Within MDCButton the default frame is 36pt tall. But, that is insufficient to meet my accessibility needs. For my application I think touch targets should be a minimum of 72pt by 72pt. The Material team has added an API to allow me to do this via `hitAreaInsets` but this does not update the `accessibilityFrame`.

## Steps to reproduce

1. Create an MDCButton
2. Set the hitAreaInsets to any value other than `UIEdgeInsetsZero`
3. Turn on VoiceOver and navigate to the button

## Expected outcome
The `accessibilityFrame` should be the size of the _hit area_.

## Actual outcome
The `accessibilityFrame` is the frame of the Button.

---
#### Internal data
- Associated internal bug: [b/134168266](http://b/134168266)

**Comment by romoore** [comment]
`hitAreaInsets` adjust the touch target but not the accessibility frame. Closing this as "works as intended" since the `hitAreaInsets` API is not intended to adjust accessibility frame.
